src/main/resources/config/defaults-storage-driver-pravega.yaml

```diff
@@ -14,6 +14,7 @@ storage:
         count: 0
       timeoutMillis: 100
     read:
+      e2eMode: false
```

Comment:
Please, update the [4.1. Specific Options](https://github.com/emc-mongoose/mongoose-storage-driver-pravega#41-specific-options) considering this parameter.

Reply:
nice catch, done